addons/overpressure/stringtable.xml

```diff
@@ -2,7 +2,7 @@
 <Project name="ACE">
     <Package name="Overpressure">
         <Key ID="STR_ACE_Overpressure_DisplayName">
-            <English>Overpressure</English>
+            <English>Overpressure and Backblast</English>
```

Comment:
I'm going to revert this for now, as it invalidates the current stringtable entries, but we can revisit the idea later imo.